v2/internal/controllers/crd_servicebus_namespace_standard_test.go

```diff
@@ -97,4 +103,50 @@ func ServiceBus_Topic_CRUD(tc *testcommon.KubePerTestContext, sbNamespace client
 	// a basic assertion on a property
 	tc.Expect(topic.Status.SizeInBytes).ToNot(BeNil())
 	tc.Expect(*topic.Status.SizeInBytes).To(Equal(0))
+
+	tc.RunParallelSubtests(
+		testcommon.Subtest{
+			Name: "Namespace secrets",
+			Test: func(tc *testcommon.KubePerTestContext) { ServiceBus_Subscription_CRUD(tc, topic) },
+		},
+	)
+}
+
+func ServiceBus_Subscription_CRUD(tc *testcommon.KubePerTestContext, sbTopic client.Object) {
+	subscription := &servicebus.NamespacesTopicsSubscription{
+		ObjectMeta: tc.MakeObjectMeta("subscription"),
+		Spec: servicebus.Namespaces_Topics_Subscription_Spec{
+			Location: tc.AzureRegion,
+			Owner:    testcommon.AsOwner(sbTopic),
+		},
+	}
+	tc.ExportAsSample(subscription)
+
+	tc.CreateResourceAndWait(subscription)
+	defer tc.DeleteResourceAndWait(subscription)
```

Comment:
minor: Rather than doing `defer` here for deletion, consider doing:

```
	tc.DeleteResourceAndWait(namespace)
	// Ensure that the resource was really deleted in Azure
	tc.ExpectResourceIsDeletedInAzure(armId, string(servicebus.APIVersion_Value))
```

At the end, as this ensures that the Delete part of CRUD really works